src/data/formatters.test.js

```diff
@@ -55,38 +55,51 @@ describe("formatPrice", () => {
   });
 });
 
-describe("formattedEventPrice", () => {
-  it("returns Free as the price when isFree is true", () => {
-    const result = formattedEventPrice(true, 0, 0);
+describe("formattedShortEventPrice", () => {
+  it("returns Free as the price when both prices are zero", () => {
+    const result = formattedShortEventPrice(0, 0);
     expect(result).toBe("Free");
   });
-  it("returns the eventPriceLow when it equals the eventPriceHigh and isFree is false", () => {
-    const result = formattedEventPrice(false, 4.55, 4.55);
+
+  it("returns the eventPriceLow when it equals the eventPriceHigh", () => {
+    const result = formattedShortEventPrice(4.55, 4.55);
     expect(result).toBe("£4.55");
   });
+
   it("returns From eventPriceLow when there is a range of prices", () => {
-    const result = formattedEventPrice(false, 12, 15.5);
+    const result = formattedShortEventPrice(12, 15.5);
     expect(result).toBe("From £12");
   });
+
+  it("returns From eventPriceLow when there is a range of prices from £0", () => {
+    const result = formattedShortEventPrice(0, 15.5);
+    expect(result).toBe("From £0");
+  });
+
+  it("returns eventPriceLow when the prices are the same, but not zero", () => {
+    const result = formattedShortEventPrice(15.5, 15.5);
+    expect(result).toBe("£15.50");
+  });
 });
 
-describe("formattedEventPriceRange", () => {
+describe("formattedLongEventPrice", () => {
   it("returns free for free events", () => {
-    expect(formattedEventPriceRange(true, 1, 2)).toEqual("Free");
+    expect(formattedLongEventPrice(0, 0)).toEqual("Free");
+  });
+
+  it("returns the range of prices when zero and a higher price are given", () => {
+    expect(formattedLongEventPrice(0, 2)).toEqual("£0 - £2");
```

Comment:
The separator here should be an en-dash `–`, not a hyphen `-`.

Reply:
👍